### PR TITLE
Fix AppVeyor build

### PR DESCRIPTION
**Pull request by julienfalque**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ install:
     - echo extension=php_openssl.dll >> php.ini
     - echo extension=php_mbstring.dll >> php.ini
     - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer-stable.phar)
-    - rename C:\tools\composer-stable.phar composer.phar
+    - rename composer-stable.phar composer.phar
```
Comment (guilliamxavier) [inline .appveyor.yml:33]: Should that second line be merged withing the first? i.e.:
```yml
     - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer-stable.phar && rename composer-stable.phar composer.phar)
```

Comment (julienfalque) [reply .appveyor.yml:33]: That's the fix I intend to push, just checking :)

Comment (guilliamxavier) [reply .appveyor.yml]: Apparently you can even simplify
`appveyor DownloadFile https://getcomposer.org/composer-stable.phar && rename composer-stable.phar composer.phar` into
`appveyor DownloadFile https://getcomposer.org/composer-stable.phar -FileName composer.phar`

Comment (julienfalque) [reply .appveyor.yml]: Nice, thanks!

**Pull request merged**